doc/content/devices/models/rakwireless-rak2560/_index.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "RAKwireless RAK2560 Sensor Hub"
+description: ""
+weight: 
+---
+
+{{< figure src="rak2560.png" alt="RAK2560" class="float plain" width="80%">}}
+
+The RAKwireless [RAK2560 WisNode Sensor Hub](https://store.rakwireless.com/products/sensor-hub) is a modular sensor ecosystem consisting of the Sensor Hub Smart Datalogger and multiple pre-configured sensor probes. With pluggable, interchangeable probes, and the option to add third-party sensors to the mixture, the Sensor Hub is a suitable and versatile solution platform for various IoT applications where environmental monitoring is needed. 
+
+<!--more-->
+
+## Sensor Setup
+
+To acquire the provisioning information and properly setup the sensor you must download the **WisToolBox** App from the Play Store or the App Store on your phone.
+
+After opening the app it will ask you to log in/create an account, but you can click on **Skip** in the top right if you don't wish to sign up.
+
+Then press **Start**, select **NFC Activation** and follow the instructions for connecting to the device.
+
+Once you reach the menu of the device, select the **LORA & LORAWAN PARAMETERS** tab. After that open the **Global Settings** dropdown and make sure the Join mode and Region are set correctly. 
+
+Then open the **LoRaWAN keys, ID, EUI** dropdown and copy the **Device EUI**, **AppEUI** and **AppKey** and save them somewhere as they will be needed later for onboarding.
+
+{{< figure src="wistoolbox.png" alt="Settings tab of the device." width="40%">}}
+
+Finally select the **Sensor Probe** tab up top and open the dropdowns for the respective sensors that are used. Here you can set the intervals or thresholds of when to send data.
+
+Make sure you enable the **Send uplink** toggle depending on which method  you use.
```

Comment:
```suggestion
Make sure you enable the **Send uplink** toggle depending on which method you use.
```

Reply:
In the app you can select either "Send periodic uplink" or "Send uplink if below threshold" or "Send uplink if above threshold" or "Send uplink if between threshold", so theres multiple methods of when to send data.